pkg/cli/images_rm.go

```diff
@@ -43,7 +42,7 @@ func (a *ImageDelete) Run(cmd *cobra.Command, args []string) error {
 		}
 
 		if img == nil && len(removedTags) == 0 {
-			logrus.Infof("No image found for %s", image)
+			fmt.Printf("No image found for %s", image)
```

Comment:
Why the switch from logrus?

Reply:
fmt all around here, since it's CLI output

Reply:
Ahhh I see!